src/tool_fsharp/tool_fsc.csproj

```diff
@@ -0,0 +1,70 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.tasks" />
+
+  <PropertyGroup>
+    <VersionPrefix>$(CliVersionPrefix)</VersionPrefix>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RuntimeFrameworkVersion>$(CLI_SharedFrameworkVersion)</RuntimeFrameworkVersion>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <PublishDir>$(FSharpDirectory)</PublishDir>
+    <VersionSuffix>$(CommitCount)</VersionSuffix>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NetCore.App"                Version="$(CLI_SharedFrameworkVersion)" />
+    <PackageReference Include="Microsoft.FSharp.Core.netcore"        Version="$(CLI_FSharp_Version)" />
+    <PackageReference Include="Microsoft.FSharp.Compiler.netcore"    Version="$(CLI_FSharp_Version)" />
```

Comment:
Nit: don't bother lining up Version, some editors will undo the change and cause noise in future PRs.